src/geojson/geojson.ml

```diff
@@ -28,6 +28,12 @@ module Make (J : Intf.Json) = struct
     Option.(
       if is_some bbox then [ ("bbox", J.array J.float (get bbox)) ] else [])
 
+  let foreign_members_or_empty foreign_members =
+    Option.(
+      if is_some foreign_members then
+        [ ("foreign_members", get foreign_members) ]
```

Comment:
> Note, however, there is no JSON field called "foreign_members". The "foreign_members" constitute anything else that may be left in the object after trying to parse it.

From a previous review (see https://github.com/geocaml/ocaml-geojson/pull/45#discussion_r889371331). We never convert the foreign members out of the JSON representation.

```suggestion
        get foreign_members
```